Lib/test/test_compile.py

```diff
@@ -43,7 +43,6 @@ def test_debug_assignment(self):
         setattr(builtins, '__debug__', prev)
 
     # TODO: RUSTPYTHON
-    @unittest.expectedFailure
```

Comment:
When you remove any decorators like this, please also remove the corresponding `TODO: RUSTPYTHON` comment.

Reply:
Thanks, updated